include/onnxruntime/core/providers/dnnl/dnnl_provider_options.h

```diff
@@ -0,0 +1,8 @@
+// Copyright(C) 2022 Intel Corporation
+// Licensed under the MIT License
+#pragma once
+
+struct OrtDnnlProviderOptions {
+  int use_arena;          // If arena is used, use_arena 0 = not used, nonzero = used
+  void* threadpool_args;  // Used to enable ORT threadpool when using the test runner
```

Comment:
Comment needs an update: Why only when using the test runner? Given that this is exposed in the API, it doesn't seem limited to the test runner.

Reply:
Fixed, thank you!